src/pvuncertainty.cpp

```diff
@@ -114,13 +114,13 @@ PVUncertaintyForm::PVUncertaintyForm( wxWindow *parent, Case *cc )
     std::vector< std::tuple<std::string, std::string, std::string > > sourceinfo;
-	sourceinfo.push_back(std::make_tuple("Irradiance transposition", "Uncertainty in irradiance transposition from horizontal to plane of array. Consider whether\nthe irradiance data in the weather file is plane-of-array or horizontal, and whether input irradiance\ncomponents are DNI+DHI, DNI+GHI, or GHI+DHI. See inputs on Location and Resource page,\nincluding options under Albedo - Sky Diffuse Model - Irradiance Data (Advanced) at the bottom\nof the page.", "Irradiance transposition:1:0:0:0:0"));
+	sourceinfo.push_back(std::make_tuple("Irradiance transposition", "Uncertainty in irradiance transposition from horizontal to plane of array. Consider whether\nthe irradiance data in the weather file is plane-of-array or horizontal, and whether input irradiance\ncomponents are DNI+DHI, DNI+GHI, or GHI+DHI. See inputs on Location and Resource page,\nincluding options under Albedo - Sky Diffuse Model - Irradiance Data (Advanced) at the bottom\nof the page.", "Irradiance transposition:1:0:0.1:0:0"));
 	sourceinfo.push_back(std::make_tuple("Horizon shading", "Uncertainty in horizon shading. Consider whether solar irradiance data in the weather file accounts\nfor horizon shading.", "Horizon shading:4:-1:0:0:0"));
 	sourceinfo.push_back(std::make_tuple("Row shading", "Uncertainty in row-to-row shading (self shading). Consider whether self shading is enabled on the Shading and Layout page.", "Row shading:4:-5:-1:0:0"));
 	sourceinfo.push_back(std::make_tuple("Single module rating at STC", "Uncertainty of STC power rating for a single module of the type defined on the Module page.", "Single module rated power at STC:1:0:2:0:0"));
 	sourceinfo.push_back(std::make_tuple("Inverter availability", "Uncertainty in inverter downtime for maintenance or replacement. Consider the type of inverter defined on\nthe Inverter page and any assumptions for inverter operating costs on the Operating Costs page.", "Inverter availability:4:-5.7:-2.7:0:0"));
-	sourceinfo.push_back(std::make_tuple("Spectral response", "Uncertainty in differences between the reference spectral shape of incident irradiance at STC during\nindoor tests for module rating and outdoor operating conditions.", "Spectral response:1:0:0:0:0"));
-	sourceinfo.push_back(std::make_tuple("Cell temperature", "Uncertainty in measurement of the module temperature coefficient of power. Consider the parameters on the Module page.", "Cell temperature:1:0:0:0:0"));
+	sourceinfo.push_back(std::make_tuple("Spectral response", "Uncertainty in differences between the reference spectral shape of incident irradiance at STC during\nindoor tests for module rating and outdoor operating conditions.", "Spectral response:1:0:0.1:0:0"));
+	sourceinfo.push_back(std::make_tuple("Cell temperature", "Uncertainty in measurement of the module temperature coefficient of power. Consider the parameters on the Module page.", "Cell temperature:1:0:0.1:0:0"));
```

Comment:
Thanks for updating these - at some point we should revisit moving them to the defaults.